client/components/EventsListContainer/index.jsx

```diff
@@ -75,7 +75,8 @@ class EventsListComponent extends React.Component {
                                 loadMoreEvents={this.props.loadMoreEvents}
                                 selectedEvents={this.props.selectedEvents}
                                 onEventSelectChange={this.props.onEventSelectChange}
-                                privileges={privileges} />
+                                privileges={privileges}
+                                session={session} />
```

Comment:
I see.
we are now passing too many props to `EventItem` from `EventListContainer` and `EventList` which don't need them.
We should maybe start to connect `EventItem` or a new container of it in order to avoid this in the future.